test/acceptance/mock-proxy/index.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+var config = require('../../../config');
+var _ = require('underscore');
+var express = require('express');
+var httpProxy = require('http-proxy')
+  .createServer({
+    target:
+      'http://' +
+      config.api.real.host +
+      ':' +
+      config.api.real.port
+  });
+
+require('express-hijackresponse');
+
+var app = express();
+var willReturn = 1;
+
+app.use(function hijackRes(req, response, next) {
+  response.hijack(function (err, res) {
+    if (err) {
+      res.unhijack();
+      return next(err);
+    }
+
+    if (/\/json$/.test(res.getHeader('Content-Type'))) {
+      res.removeHeader('Content-Length');
+      res.writeHead(res.statusCode);
+
+      res.body = '';
+
+      res.on('data', function (chunk) {
+          res.body += chunk;
+      }).on('end', function () {
+          var events = JSON.parse(res.body);
```

Comment:
It's a good idea to `JSON.parse` inside a `try catch` so you can throw an error if the JSON object cannot be parsed.